.circleci/config.yml

```diff
@@ -16,6 +16,15 @@ aliases:
 
 jobs:
   lint:
+    executor:
```

Comment:
Is it worth distinguishing `lint` from `vet`? Perhaps make them the same job?

Reply:
I did consider that, but I like having them as separate checks so that you can immediately tell what failed in GitHub. Also, CircleCi seems to support this kind of thing pretty well and this separation is in line with recommendations for building gitlab-ci pipelines (which have worked well for me in the past).

I propose that we try this and change in the future if we find a reason to do so.